src/Components/Facility/DischargeModal.tsx

```diff
@@ -95,6 +99,15 @@ const DischargeModal = ({
     setFacility(referred_to);
   }, [referred_to]);
 
+  const { data } = useQuery(routes.getConsultation, {
+    pathParams: {
+      id: consultationData.id,
+    },
+  });
+
+  const ConsultationDiagnosisList: ConsultationDiagnosis[] =
+    data?.diagnoses?.map((diagnosis) => diagnosis) || [];
```

Comment:
Why is there an unnecessary `.map` call?
```suggestion
    data?.diagnoses ?? [];
```

Reply:
My bad, i was return something from each diagnosis item to populate into new array. Forget to change it later. Commited as suggested now. Thanks